pkgs/applications/audio/mopidy/default.nix

```diff
@@ -12,16 +12,19 @@ pythonPackages.buildPythonPackage rec {
     sha256 = "1xfyg8xqgnrb98wx7a4fzr4vlzkffjhkc1s36ka63rwmx86vqhyw";
   };
 
+  buildInputs = [
+    wrapGAppsHook gst_plugins_base gst_plugins_good gst_plugins_ugly glib_networking
+  ];
+
   propagatedBuildInputs = with pythonPackages; [
-    gst_python pygobject pykka tornado requests2 gst_plugins_base gst_plugins_good gst_plugins_ugly
+    gst_python pygobject pykka tornado requests2
   ];
 
   # There are no tests
   doCheck = false;
 
-  postInstall = ''
-    wrapProgram $out/bin/mopidy \
-      --prefix GST_PLUGIN_SYSTEM_PATH : "$GST_PLUGIN_SYSTEM_PATH"
+  preFixup = ''
+    gappsWrapperArgs+=(--prefix GST_PLUGIN_SYSTEM_PATH : "$GST_PLUGIN_SYSTEM_PATH")
```

Comment:
It would be valuable if you modified wrapGAppsHook to include `GST_PLUGIN_SYSTEM_PATH` automatically as it's done for gst 1.0.